browser/src/control/Control.JSDialog.js

```diff
@@ -81,6 +81,10 @@ L.Control.JSDialog = L.Control.extend({
 		return builder;
 	},
 
+	getDialog: function (id) {
```

Comment:
Please do not use that data outside this class. It's very important.

Reply:
https://github.com/CollaboraOnline/online/pull/12232/files#r2175148896

It's implementation detail which shouldn't leak.